src/components/vanilla/charts/CompareLineChart/CompareLineChart.emb.ts

```diff
@@ -156,10 +152,10 @@ export default defineComponent(Component, meta, {
             granularity: inputs.granularity
           }
         ],
-        limit: !inputs.prevTimeFilter?.from ? 1 : 500,
+        limit: !inputs.prevTimeFilter ? 1 : 500,
         measures: inputs.metrics,
         filters:
-          inputs.prevTimeFilter?.from && inputs.xAxis
```

Comment:
@mad-raccoon did you test this? I think the prevTimeFilter needs a .from, since a relativeTimeString won't work because the dates for the compared period have to be calculated based on the values of the regular filter.

Reply:
@hjkmarshall i tested, the compared data can be a relativeTimeString. i provided evidence for it